eng/Packages.Data.props

```diff
@@ -314,6 +314,8 @@
     Test/Sample/Perf/Stress project dependencies
   -->
   <ItemGroup Condition="('$(IsTestProject)' == 'true') OR ('$(IsTestSupportProject)' == 'true') OR ('$(IsPerfProject)' == 'true') OR ('$(IsStressProject)' == 'true') OR ('$(IsSamplesProject)' == 'true')">
+    <PackageReference Update="Azure.Core.TestFramework" Version="1.0.0" />
+    <PackageReference Update="Azure.ResourceManager.TestFramework" Version="1.0.0" />
```

Comment:
These should be linked in as source.   They're not meant to be consumed as packages.    Revert this, please.